perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -730,6 +730,24 @@ sub stash_category_groups : Private {
         }
     }
 
+    # New reporting, top level mixes groups and categories not in a group
+    if ($opts->{mix_in}) {
+        my $no_group = delete $category_groups{""};
+        my @list = map { [ $_->category_display, $_ ]  } @$no_group;
+        foreach (keys %category_groups) {
+            (my $id = $_) =~ s/[^a-zA-Z]+//g;
+            push @list, [ $_, { id => $id, name => $_, categories => $category_groups{$_} } ];
```

Comment:
The same `id` key needs to be included on L759 & L761–L763, I believe, otherwise `group.id` in `subcategories.html` doesn't render correctly (eg  at `/around?pc=pier+walk%2C+greenwich`):

<img width="953" alt="image" src="https://user-images.githubusercontent.com/4776/106742442-f58d7680-6614-11eb-9d9a-9044a9716e10.png">

Reply:
Hmm, I guess on an around page it shouldn't be trying to show these at all, because they'll only be overridden as soon as you click on a map! (It's using the around map filter list, not where a report would be, too, which is confusing). When it was one select, it was within a IF js loop, so it probably should be too, which should fix this. 
So I don't think you need the ID in those lines, because it shouldn't get there with report/new list, if that makes sense.

Reply:
Have added `IF NOT js` around the include of `report/new/subcategories.html` in `form_report.html` which should fix this